public_suffix_list.dat

```diff
@@ -6584,6 +6584,17 @@ lib.wy.us
 pvt.k12.ma.us
 chtr.k12.ma.us
 paroch.k12.ma.us
+// Merit Network, Inc. maintains the registry for =~ /(k12|cc|lib).mi.us/ and the following
+//    see also: http://domreg.merit.edu
+//    see also: whois -h whois.domreg.merit.edu help
```

Comment:
Of course. We can make any of the suggested modifications.

I wrote the initial comments with the intent to articulate that Merit is the registry and solo registrar of many, but not all, of the mi.us. 3LDs. Other mi.us 3LDs are registered directly with [nic.us](http://nic.us) under their [Locality-Based Name Registrations](http://www.neustar.us/locality-based-name-registrations/) program. It is unclear if other mi.us 3LD registrations are used by single institutions or if they offer a registry for their respective memberships.

Historically, these domains have operated in a fashion more inline with the PSL's ICANN definition then the PRIVATE definition.